doc/en/components/grids/grid/groupby.md

```diff
@@ -8,7 +8,7 @@ namespace: Infragistics.Controls
 
 # {Platform} Grid Group By
 
-A Group By behavior in an {Platform} Material table or UI grid creates grouped data rows based on the column values. The Group By in the `Grid` allows for visualizing the groups in a hierarchical structure. The grouped data rows can be expanded or collapsed and the order of grouping may be changed through the UI or API. When Row Selection is enabled, a Group By row selector is rendered in the left-most area of the group row. In case the `RowSelection` property is set to single, checkboxes are disabled and only serve as an indication for the group where selection is placed. If the `RowSelection` property is set to multiple, clicking over the Group By row selector selects all records belonging to this group.
+The {ProductName} Group By behavior in {Platform} {ComponentTitle} creates grouped data rows based on the column values. The Group By in the `Grid` allows for visualizing the groups in a hierarchical structure. The grouped data rows can be expanded or collapsed and the order of grouping may be changed through the UI or API. When Row Selection is enabled, a Group By row selector is rendered in the left-most area of the group row. In case the `RowSelection` property is set to single, checkboxes are disabled and only serve as an indication for the group where selection is placed. If the `RowSelection` property is set to multiple, clicking over the Group By row selector selects all records belonging to this group.
```

Comment:
`{ComponentTitle}` work in _shared folder only, this topic is only for the flat grid so you can use IgrGrid or Grid directly
![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/1182001/84ac1a26-4c3a-4eba-b6d7-a3bbd95b9b98)